Doc/whatsnew/3.10.rst

```diff
@@ -1093,6 +1093,12 @@ linecache
 When a module does not define ``__loader__``, fall back to ``__spec__.loader``.
 (Contributed by Brett Cannon in :issue:`42133`.)
 
+math
```

Comment:
This is in the wrong "whatsnew" document; this would be a change for Python 3.11, not Python 3.10.

Reply:
will do that too.